src/main/com/example011/radek/v1/Mines.java

```diff
@@ -0,0 +1,173 @@
+package main.com.example011.radek.v1;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Scanner;
+
+public class Mines {
+
+    protected int playgroundSize;
+
+    protected int numMines;
+
+    protected String[][] playground;
+
+    public Mines(int size, int mines) {
+
+        if (setPlaygroundSize(size)) {
+            int setNumMinesResult = setNumMines(mines);
+            if (setNumMinesResult == 1) {
```

Comment:
proc kdyz je result 1 tak delej neco, co ta jednicka predstavuje? nechceme tu zadne magic number a flagy, jde to i bez toho

Reply:
Přepracováno, logika přesunuta do setNumMines, nyní je void

Reply:
setNumMines ... nvm, co si pod tim predstavit? zlepsit naming

Reply:
Přejmenováno